kex-runner/src/main/kotlin/org/vorpal/research/kex/asm/analysis/concolic/InstructionConcolicChecker.kt

```diff
@@ -139,7 +137,26 @@ class InstructionConcolicChecker(
         val testFile = generator.emit()
 
         compilerHelper.compileFile(testFile)
-        collectTrace(generator.testKlassName)
+        val result = collectTrace(generator.testKlassName)
```

Comment:
I remember talking about it, but I forgot. Why did we decide to extract this to a different coroutine scope?

Reply:
The idea was to return the result from test execution right away which may result in better coverage, but less oracles given the same generation time. And dispatchers IO used for test printing because the thread will be stopped for io operations, also small modification